src/utils/calculateClicksAndImpressions.js

```diff
@@ -1,6 +1,11 @@
+// @flow
 import _ from "lodash";
 
-const calculateClicksAndImpressions = (campaignOrChannel, data, metricName) =>
+const calculateClicksAndImpressions = (
+  campaignOrChannel: string,
+  data: Object,
```

Comment:
please introduce a type here (instead of `Object`) and re-use everywhere :exclamation:

Reply:
b00d657a1eb3c594dd24ad62b7ebd45053f7249e

Reply:
e55b4a1b7b4021f9863c371c8728b0269f330840